src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -408,17 +408,19 @@ function ProviderSelectionField({
                   </button>
                 </>
               )}
-              <button
-                type="button"
-                className="vaos-appts__cancel-btn va-button-link vads-u-margin--0 vads-u-flex--0"
-                onClick={() => {
-                  setProvidersListLength(INITIAL_PROVIDER_DISPLAY_COUNT);
-                  setShowProvidersList(false);
-                }}
-                aria-label="Cancel choosing a provider"
-              >
-                Cancel
-              </button>
+              {communityCareProviderList.length > 0 && (
```

Comment:
I don't think `communityCareProviderList` is guaranteed to exist (it's hard to tell in this component), you might need to do `communityCareProviderList?.length`

Reply:
Done.